JIT.cpp

```diff
@@ -0,0 +1,43 @@
+#include "JIT.h"
```

Comment:
Guard the inclusion of this, either with a `pragma once` or the a header ifdef.

Reply:
Wait the header is guarded though right?

Reply:
You're right, I don't know what I was thinking. Sorry for the noise.